.gitignore

```diff
@@ -62,3 +62,5 @@ typings/
 
 # IDE specific
 .idea
+*.sublime-project
+*.sublime-workspace
```

Comment:
added this so repo would ignore files for my personal IDE / text editor.  Can remove this if you don't want it.

Reply:
It's good to keep it